### DISCUSSION TITLE
Intra gNB Handover using ZMQ + GNU Radio setup?

### DISCUSSION THREAD

**Discussion by dineshpv** [srsran/srsRAN_Project#681] [General Help] [answered]
Hello all, 
I have been trying to run a scenario of Intra gNB Handover in which I am planning to get two DUs with one cell each under a gNB using ZMQ and GNU Radio (grc). I have taken srsRAN_4G ZMQ handover tutorial as my reference, but I am failing in doing so with gNB. I am facing some issues, where I see that the cells are not coming up. Are there any tutorials or discussions regarding this, please let me know It would be helpful.

**Comment by pgawlowicz** [comment, marked as answer]
The 5g UE provided in srsRAN_4G project does not support handover.
